tex/generic/pgf/systemlayer/pgfsys-common-postscript.def

```diff
@@ -1,5 +1,6 @@
 % Copyright 2019 by Till Tantau
 % CMYK and grayscale shadings adaptation copyright 2019 by David Purton
+
```

Comment:
You have moved a lot of content from `pgfsys-dvips.def` to this file.  Have you actually verified that this does not break the other drivers that use `pgfsys-common-postscript.def`?  The two other drivers are `vtex` and `textures` and are not checked by the CI.

Reply:
You are right. My additions heavily depend on new Ghostscript features. Since VTeX and probably Textures ship with their own PostScript interpreters and since it seems impossible to run tests with these ancient systems I moved all my additions back to `pgfsys-dvips.def`.  Also, for both drivers I restored the traditional way of producing shadings. Besides, I would plead for dropping support for both drivers in question. They haven't been updated for 1.5 decades and it is questionable whether they work at all with an isolated up-to-date pgf/TikZ package using old formats and binaries.

Reply:
Yes, I agree.  I will retire support for these engines, in particular because I don't think either of them supports e-TeX which is needed for recent PGF releases.